src/objdump.py

```diff
@@ -63,6 +63,33 @@ class ArchSettings:
     "bc1f",
 }.union(MIPS_BRANCH_LIKELY_INSTRUCTIONS)
 
+PPC_BRANCH_INSTRUCTIONS = {
+    "b",
+    "beq",
+    "beq+",
+    "beq-",
+    "bne",
+    "bne+",
+    "bne-",
+    "blt",
+    "blt+",
+    "blt-",
+    "ble",
+    "ble+",
+    "ble-",
+    "bdnz",
+    "bdnz+",
+    "bdnz-",
+    "bge",
+    "bge+",
+    "bge-",
+    "bgt",
+    "bgt+",
+    "bgt-",
+}
+
+PPC_BRANCH_LIKELY_INSTRUCTIONS = {}
```

Comment:
This ends up a dict instead of a set (mypy will point this out if you set it up, fine if you don't want to worry about that for now though). Can inline it as `set()` at the usage site instead.